packages/office-ui-fabric-react/src/components/MessageBar/MessageBar.styles.ts

```diff
@@ -39,19 +39,20 @@ const getRootBackground = (
   messageBarType: MessageBarType | undefined,
   palette: IPalette,
   semanticColors: ISemanticColors,
+  isInverted: boolean,
 ): string => {
   switch (messageBarType) {
     case MessageBarType.error:
     case MessageBarType.blocked:
       return semanticColors.errorBackground;
     case MessageBarType.severeWarning:
-      return semanticColors.blockingBackground;
+      return isInverted ? '#4F2A0F' : '#FED9CC';
```

Comment:
don't hardcode colors in components

Reply:
Hi @phkuo, thanks for taking a look. 

There's already a pretty strong precedent for hardcoding the colors for MessageBar. However, if you'd like me to move these colors elsewhere, or use another approach, please provide some more detailed guidance.

Reply:
Hey @chrismohr ! I have a similar [PR(stale)](https://github.com/microsoft/fluentui/pull/12267/files) open for this. You can make new slots in Semantic Slots, for example 'severeWarningBackground' to avoid hard coded values in MessageBar styles directly

Reply:
Thanks @marygans, I can add the slots.  Just curious if there's a reason you didn't move forward with your PR?  or are you still planning to?  If so, I can abandon this one.  (That said, I was hoping to get the change in this week.  We have a release next week that's depending on it, and rather than overriding the styles in our app, I thought it would be better to tackle it here.)